src/animator.js

```diff
@@ -351,6 +377,8 @@ export class VelocityAnimator {
       el.animations = {};
       el.animations.enter = this._parseAttributeValue(el.getAttribute('anim-enter')) || this.enterAnimation;
       el.animations.leave = this._parseAttributeValue(el.getAttribute('anim-leave')) || this.leaveAnimation;
+      el.animations.show = this._parseAttributeValue(el.getAttribute('anim-show')) || undefined;
```

Comment:
I initially did that, but reconsidered. Then enter/leave effects are opt-in by adding `au-animate`. If hide/show should have a default animation, I think we would also have to add a check for that attribute. (Or perhaps accept empty `anim-leave` and `anim-show` attributes.)